Automattic/jetpack
# VideoPress: Bring edit while upload from V6 and support it from Library

It brings back all changes made at #25392 from Jetpack Plugin to the VideoPress package.

And already includes the support to edit while uploading local media from Library.

Closes #25788 

#### Changes proposed in this Pull Request:

* Merge edit while uploading from Jetpack to VideoPress package.
* Support editing while uploading from Library.
* Fix `blockProps` not being applied to all instances.

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Enable `VideoPress` or `Jetpack` plugin.
* Use `VideoPress` block.
* Try to upload a new file.
* You should be able to edit while uploading.
* Try to upload a local file from the library.
* You should be able to edit while uploading.

#### Demo

_Edit while uploading from Library_


https://user-images.githubusercontent.com/1663717/186760510-55c9cd82-7fe1-4dc6-8c79-d9cc97522361.mov
